src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/RunReportPageTests.kt

```diff
@@ -102,4 +106,47 @@ class RunReportPageTests : IntegrationTest()
                 ReportWithDate("minimal", repo.getLatestVersion("minimal").date)
         )
     }
+
+    @Test
+    fun `can get running reports details`()
+    {
+        val instant = Instant.now()
+        val fakeReportRunLog = ReportRunLog(
+                "test@example.com",
+                instant,
+                "q123",
+                "{‘0’: {‘source’: ‘support’, ‘annexe’: ‘annexe val’}}",
+                "{‘0’: {‘name’: ‘cologne’, ‘value’: ‘memo’}}",
+                "branch",
+                "commit",
+                "complete",
+                "logs",
+                "1233")
+
+        val mockContext: ActionContext = mock {
+            on { params(":key") } doReturn "fakeKey"
+        }
+
+        val mockServer: OrderlyServerAPI = mock {
+            on { get("/running/fakeKey/logs", mockContext) } doReturn
+                    OrderlyServerResponse(Serializer.instance.toResult(fakeReportRunLog), 200)
+        }
+
+        val mockRepo = mock<ReportRepository> {
+            on { getReportRun("fakeKey") } doReturn fakeReportRunLog
+        }
+
+        val sut = ReportController(mockContext, mock(), mockServer, mockRepo, mock())
+        val result = sut.getRunningReportsDetails()
+        assertThat(result.email).isEqualTo("test@example.com")
```

Comment:
Any reason why this isn't just:
```
assertThat(result).isEqualTo(fakeReportRunLog)
```
That would be legitimate, given the object has been serialized to string and back again.